webapp/components/form/AutocompleteDialog/store/state/state.js

```diff
@@ -24,12 +26,19 @@ export const getList = A.prop(keys.list)
 
 export const getItemsSize = (state) => getItems(state).length
 
+const _calculateVerticalPosition = ({ top: elementTop, height: elementHeight }) => {
+  const top = elementTop + elementHeight
+  const windowScreenY = window.innerHeight
+
+  return top + MAX_HEIGHT_AUTOCOMPLETE_DIAOLOG > windowScreenY ? { bottom: windowScreenY - elementTop } : { top }
+}
```

Comment:
does this open the dropdown on top?
if yes, then it shouldn't. update body height

Reply:
because users aren't used to that